app/models/course_user.rb

```diff
@@ -35,6 +35,14 @@ def experience_points
 
   delegate :experience_points, to: :experience_points_records
 
+  # Computes the level number of the CourseUser with respect to
+  # a course's Course::Levels.
+  #
+  # @return [Fixnum] CourseUser level number
+  def level_number
```

Comment:
Ok, re-ordered. Thanks for the feedback. I'd better re-read the style guide before each PR. haha